### ISSUE TITLE
Incompatible with lineno package

### ISSUE THREAD

**Issue by cfroccajr** [AndyClifton/accessibility#48]
When I try to load both accessibility and lineno I get this error:

`! LaTeX Error: Command \c@linenumber already defined.
               Or name \end... illegal, see p.192 of the manual.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.671   \newcounter{linenumber}
                               %
Your command was ignored.
Type  I <command> <return>  to replace it with another command,
or  <return>  to continue without it.`

The error varies depending on which package I load first.

**Comment by kevinwortman** [comment]
I'm hitting this too. It seems like it can be solved by changing the variable name "linecounter" to something else at https://github.com/AndyClifton/accessibility/blob/046a20886e2879e4125f7d49e3e7cc8d5dbaae9e/source/v2x/accessibility.dtx#L1160 . Are you open to a pull request?

**Comment by car222222** [comment]
Please note the following news entry:

- 23 Feb 2021: The question of accessibility in PDFs has now been taken up by the LaTeX project in a multiyear project and as a result **I am stopping all work on this package and it should be considered "not maintained"**. See https://www.latex-project.org/publications/indexbytopic/pdf/ for news about progress by the LaTeX project.

**Comment by AndyClifton** [comment]
> I'm hitting this too. It seems like it can be solved by changing the variable name "linecounter" to something non-conflicting at
> 
> https://github.com/AndyClifton/accessibility/blob/046a20886e2879e4125f7d49e3e7cc8d5dbaae9e/source/v2x/accessibility.dtx#L1160
> 
> 
> Are you open to a pull request?

Hi - yes, @kevinwortman, a pull request would be welcome, at least to record the potential solution.

**Comment by AndyClifton** [comment]
@pelinski - sorry, totally overlooked the PR and have just approved. Could you check out the new master please and see if all is OK?
